syft/frameworks/torch/tensors/interpreters/native.py

```diff
@@ -110,7 +110,7 @@ def __repr__(self) -> str:
 
             big_repr = False
 
-            if self.tags is not None:
+            if self.tags is not None and len(self.tags) > 0:
```

Comment:
Theoretically this could just be and len(self.tags) since if it is len 0 then it evaluates to false